### PR TITLE
Follow/unfollow via ajax

### PR DESCRIPTION
**Pull request by davidpdrsn**
Fixes https://github.com/davidpdrsn/CardistryIO/issues/218

@bendobos please give this a try and let me know if the UX is cool. [Screencast demo](https://dl.dropboxusercontent.com/u/2155997/follow_unfollow_ajax.mp4).

### CODE REVIEW

**Comment by bendobos** [comment]
This looks really cool! 

We could try switching out the icon with the loading spinner, to make it really clear that there is something going on. I'm thinking of users with a slow connection who may have to wait a few seconds.

**Comment by davidpdrsn** [comment]
I think just making it dim will be clear enough. I just noticed github does the same when you mark a notification as read.

**Comment by davidpdrsn** [comment]
I believe turbolinks 5 comes with a progress bar. We might be able to use that when I've added turbolinks.
